<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>331</width>
    <height>612</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">	background: &quot;#024873&quot;;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="result">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>23</y>
      <width>331</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">	qproperty-alignment: &quot;AlignVCenter | AlignRight&quot;;
	color: &quot;white&quot;;
	background-color: &quot;#04BFAD&quot;;
    font-family: 'Open Sans', sans-serif;
	font-size: 40px;
    font-weight: 600;          /* Semibold */
    line-height: 60px;         /* Межстрочный интервал */
    letter-spacing: 0.5px;     /* Межбуквенный интервал */
	border-radius: 36px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="input">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>331</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">	qproperty-alignment: &quot;AlignVCenter | AlignRight&quot;;
	color: &quot;white&quot;;
	background-color: &quot;#04BFAD&quot;;
    font-family: 'Open Sans', sans-serif;
    font-weight: 600;          /* Semibold */
    font-size: 20px;           /* Размер шрифта */
    line-height: 30px;         /* Межстрочный интервал */
    letter-spacing: 0.5px;     /* Межбуквенный интервал */</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_bracket">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 33px;
background: &quot;#0889A6&quot;;
    font-family: 'Open Sans', sans-serif;
    font-weight: 600;          /* Semibold */
    font-size: 24px;           /* Размер шрифта */
    line-height: 30px;         /* Межстрочный интервал */
    letter-spacing: 1px;     /* Межбуквенный интервал */</string>
    </property>
    <property name="text">
     <string>( )</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sign">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>170</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 33px;
background: &quot;#0889A6&quot;;
    font-family: 'Open Sans', sans-serif;
    font-weight: 600;          /* Semibold */
    font-size: 24px;           /* Размер шрифта */
    line-height: 30px;         /* Межстрочный интервал */
    letter-spacing: 1px;     /* Межбуквенный интервал */</string>
    </property>
    <property name="text">
     <string>±</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_persent">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>170</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 33px;
background: &quot;#0889A6&quot;;
    font-family: 'Open Sans', sans-serif;
    font-weight: 600;          /* Semibold */
    font-size: 24px;           /* Размер шрифта */
    line-height: 30px;         /* Межстрочный интервал */
    letter-spacing: 1px;     /* Межбуквенный интервал */</string>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_div">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>170</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 33px;
background: &quot;#0889A6&quot;;
    font-family: 'Open Sans', sans-serif;
    font-weight: 600;          /* Semibold */
    font-size: 24px;           /* Размер шрифта */
    line-height: 30px;         /* Межстрочный интервал */
    letter-spacing: 1px;     /* Межбуквенный интервал */</string>
    </property>
    <property name="text">
     <string>÷</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>260</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 33px;
background: &quot;#B0D1D8&quot;;
    font-family: 'Open Sans', sans-serif;
    font-weight: 600;          /* Semibold */
    font-size: 24px;           /* Размер шрифта */
    line-height: 30px;         /* Межстрочный интервал */
    letter-spacing: 1px;     /* Межбуквенный интервал */</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>260</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 33px;
background: &quot;#B0D1D8&quot;;
    font-family: 'Open Sans', sans-serif;
    font-weight: 600;          /* Semibold */
    font-size: 24px;           /* Размер шрифта */
    line-height: 30px;         /* Межстрочный интервал */
    letter-spacing: 1px;     /* Межбуквенный интервал */</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 33px;
background: &quot;#B0D1D8&quot;;
    font-family: 'Open Sans', sans-serif;
    font-weight: 600;          /* Semibold */
    font-size: 24px;           /* Размер шрифта */
    line-height: 30px;         /* Межстрочный интервал */
    letter-spacing: 1px;     /* Межбуквенный интервал */</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_mult">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>260</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 33px;
background: &quot;#0889A6&quot;;
    font-family: 'Open Sans', sans-serif;
    font-weight: 600;          /* Semibold */
    font-size: 24px;           /* Размер шрифта */
    line-height: 30px;         /* Межстрочный интервал */
    letter-spacing: 1px;     /* Межбуквенный интервал */</string>
    </property>
    <property name="text">
     <string>×</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>350</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 33px;
background: &quot;#B0D1D8&quot;;
    font-family: 'Open Sans', sans-serif;
    font-weight: 600;          /* Semibold */
    font-size: 24px;           /* Размер шрифта */
    line-height: 30px;         /* Межстрочный интервал */
    letter-spacing: 1px;     /* Межбуквенный интервал */
</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>350</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 33px;
background: &quot;#B0D1D8&quot;;
    font-family: 'Open Sans', sans-serif;
    font-weight: 600;          /* Semibold */
    font-size: 24px;           /* Размер шрифта */
    line-height: 30px;         /* Межстрочный интервал */
    letter-spacing: 1px;     /* Межбуквенный интервал */</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 33px;
background: &quot;#B0D1D8&quot;;
    font-family: 'Open Sans', sans-serif;
    font-weight: 600;          /* Semibold */
    font-size: 24px;           /* Размер шрифта */
    line-height: 30px;         /* Межстрочный интервал */
    letter-spacing: 1px;     /* Межбуквенный интервал */</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sub">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>350</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 33px;
background: &quot;#0889A6&quot;;
    font-family: 'Open Sans', sans-serif;
    font-weight: 600;          /* Semibold */
    font-size: 24px;           /* Размер шрифта */
    line-height: 30px;         /* Межстрочный интервал */
    letter-spacing: 1px;     /* Межбуквенный интервал */</string>
    </property>
    <property name="text">
     <string>–</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>440</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 33px;
background: &quot;#B0D1D8&quot;;
    font-family: 'Open Sans', sans-serif;
    font-weight: 600;          /* Semibold */
    font-size: 24px;           /* Размер шрифта */
    line-height: 30px;         /* Межстрочный интервал */
    letter-spacing: 1px;     /* Межбуквенный интервал */</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>440</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 33px;
background: &quot;#B0D1D8&quot;;
    font-family: 'Open Sans', sans-serif;
    font-weight: 600;          /* Semibold */
    font-size: 24px;           /* Размер шрифта */
    line-height: 30px;         /* Межстрочный интервал */
    letter-spacing: 1px;     /* Межбуквенный интервал */</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 33px;
background: &quot;#B0D1D8&quot;;
    font-family: 'Open Sans', sans-serif;
    font-weight: 600;          /* Semibold */
    font-size: 24px;           /* Размер шрифта */
    line-height: 30px;         /* Межстрочный интервал */
    letter-spacing: 1px;     /* Межбуквенный интервал */</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_add">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>440</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 33px;
background: &quot;#0889A6&quot;;
    font-family: 'Open Sans', sans-serif;
    font-weight: 600;          /* Semibold */
    font-size: 24px;           /* Размер шрифта */
    line-height: 30px;         /* Межстрочный интервал */
    letter-spacing: 1px;     /* Межбуквенный интервал */</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_0">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>530</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 33px;
background: &quot;#B0D1D8&quot;;
    font-family: 'Open Sans', sans-serif;
    font-weight: 600;          /* Semibold */
    font-size: 24px;           /* Размер шрифта */
    line-height: 30px;         /* Межстрочный интервал */
    letter-spacing: 1px;     /* Межбуквенный интервал */</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_dot">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>530</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 33px;
background: &quot;#B0D1D8&quot;;
    font-family: 'Open Sans', sans-serif;
    font-weight: 600;          /* Semibold */
    font-size: 24px;           /* Размер шрифта */
    line-height: 30px;         /* Межстрочный интервал */
    letter-spacing: 1px;     /* Межбуквенный интервал */</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_clear">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>530</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 33px;
background: &quot;#F25E5E&quot;;
font-size: 30px;
    font-family: 'Open Sans', sans-serif;
    font-weight: 600;          /* Semibold */
    font-size: 24px;           /* Размер шрифта */
    line-height: 30px;         /* Межстрочный интервал */
    letter-spacing: 1px;     /* Межбуквенный интервал */</string>
    </property>
    <property name="text">
     <string>😫</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_equals">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>530</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 33px;
background: &quot;#0889A6&quot;;
    font-family: 'Open Sans', sans-serif;
    font-weight: 600;          /* Semibold */
    font-size: 24px;           /* Размер шрифта */
    line-height: 30px;         /* Межстрочный интервал */
    letter-spacing: 1px;     /* Межбуквенный интервал */</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
